docs/framework/performance/contention-etw-events.md

```diff
@@ -5,7 +5,7 @@ ms.date: "03/30/2017"
 helpviewer_keywords: 
   - "contention events [.NET Framework]"
   - "ETW, contention events (CLR)"
-ms.assetid: 6933e753-2f2a-425b-ae84-42138c957d76
```

Comment:
We could also add a pattern for framework/performance/*-etw-events.md in the docfx.json file.

Reply:
Thanks, I'll do that.